homeassistant/components/meteoclimatic/weather.py

```diff
@@ -71,27 +75,22 @@ def condition(self):
         return format_condition(self.coordinator.data["weather"].condition)
 
     @property
-    def temperature(self):
+    def native_temperature(self):
         """Return the temperature."""
         return self.coordinator.data["weather"].temp_current
 
-    @property
-    def temperature_unit(self):
-        """Return the unit of measurement."""
-        return TEMP_CELSIUS
-
     @property
     def humidity(self):
```

Comment:
Doesn't humidity need to get the 'native_' treatment as well?

Reply:
Humidity is an percentage (%), it cannot be converted to anything else.